go/host/db/batches.go

```diff
@@ -0,0 +1,112 @@
+package db
+
+import (
+	"bytes"
+	"math/big"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/obscuronet/go-obscuro/go/common"
+	"github.com/obscuronet/go-obscuro/go/common/log"
+)
+
+// DB methods relating to batches.
+
+// headerKey = batchHeaderPrefix  + hash
+func batchHeaderKey(hash gethcommon.Hash) []byte {
```

Comment:
Style pref: typically private methods go after the public methods.

Reply:
Fixed